modules/sophus/1.22.10/patches/add_build_file.patch

```diff
@@ -0,0 +1,39 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,36 @@
++""" Builds Sophus.
++"""
++
++load("@rules_cc//cc:defs.bzl", "cc_library")
++
++cc_library(
++    name = "sophus",
++    hdrs = [
++        "sophus/average.hpp",
++        "sophus/cartesian.hpp",
++        "sophus/ceres_local_parameterization.hpp",
++        "sophus/ceres_manifold.hpp",
++        "sophus/ceres_typetraits.hpp",
++        "sophus/common.hpp",
++        "sophus/geometry.hpp",
++        "sophus/interpolate.hpp",
++        "sophus/interpolate_details.hpp",
++        "sophus/num_diff.hpp",
++        "sophus/rotation_matrix.hpp",
++        "sophus/rxso2.hpp",
++        "sophus/rxso3.hpp",
++        "sophus/se2.hpp",
++        "sophus/se3.hpp",
++        "sophus/sim2.hpp",
++        "sophus/sim3.hpp",
++        "sophus/sim_details.hpp",
++        "sophus/so2.hpp",
++        "sophus/so3.hpp",
++        "sophus/spline.hpp",
++        "sophus/types.hpp",
++        "sophus/velocities.hpp"
++    ],
++    visibility = ["//visibility:public"],
++    deps = [ "@eigen" ],
```

Comment:
@d-vo As common.hpp is using fmt https://github.com/strasdat/Sophus/blob/de0f8d3d92bf776271e16de56d1803940ebccab9/sophus/common.hpp#L36, it will be great to add `"@fmt"` to deps as well.

Reply:
We cannot modify existing versions. Feel free to submit a patch version for this, you can name it `1.22.10.bcr.1` with patch-only changes.

Reply:
@meteorcloudy I have submitted a patch https://github.com/bazelbuild/bazel-central-registry/pull/1495. How to add "skip-url-stability-check" label in the PR as [the repo](https://github.com/strasdat/Sophus) does not provide link in the desired format?